content/en/docs/concepts/windows/user-guide.md

```diff
@@ -215,19 +222,20 @@ tolerations:
 ### Handling multiple Windows versions in the same cluster
 
 The Windows Server version used by each pod must match that of the node. If you want to use multiple Windows
-Server versions in the same cluster, then you should set additional node labels and nodeSelectors.
+Server versions in the same cluster, then you should set additional node labels and `nodeSelector` fields`.
 
-Kubernetes 1.17 automatically adds a new label `node.kubernetes.io/windows-build` to simplify this.
-If you're running an older version, then it's recommended to add this label manually to Windows nodes.
+Kubernetes automatically adds a label,
+[`node.kubernetes.io/windows-build`](/docs/reference/labels-annotations-taints/#nodekubernetesiowindows-build)
+to simplify this.
 
 This label reflects the Windows major, minor, and build number that need to match for compatibility.
-Here are values used today for each Windows Server version.
+Here are values used for each Windows Server version:
 
-| Product Name                         |   Build Number(s)      |
-|--------------------------------------|------------------------|
-| Windows Server 2019                  | 10.0.17763             |
-| Windows Server, Version 20H2         | 10.0.19042             |
-| Windows Server 2022                  | 10.0.20348             |
+| Product Name                         | Label value(s)       |
```

Comment:
20348 is a build number; is 10.0.20348 also a build number?

Reply:
Windows builds have 4 parts `major`.`minor`.`revision`.`patch`.
Since Windows 10 / Windows Server 2019 the `major`.`minor` has been locked at 10.0.
The `revision` can be used to identify the release (Windows Server 2019 vs Windows Server 2022) and the `patch` can be used to identify which set of cumulative patches are applied.
Windows containers are compatible across patch versions for any revision of Windows.

Reply:
OK, but the question to answer is what we call a `major`.`minor`.`revision`.`patch` in the table header.

If it's a “build number”, then https://learn.microsoft.com/en-us/windows/release-health/release-information is wrong to call `revision`.`patch` a build number.

If it's not a “build number”, tell me the right header text.

Reply:
Let's stick with build number.

In the Windows registry this value is reported as the `CurrentBuildNumber` and we query this data through the registry.

This is what my Windows 11 machine shows...

```cmd
 C:\Users\marosset>reg query "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion" /v CurrentBuildNumber

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion
    CurrentBuildNumber    REG_SZ    22621


C:\Users\marosset>cmd /ver
Microsoft Windows [Version 10.0.22621.1555]
```

Reply:
I don't get it. If `22621` is the build number than `10.0.22621.1555` _isn't_ the build number.

Surely someone at Microsoft knows a formal name for the longer string with the periods in it?